airbyte-webapp/src/locales/en.json

```diff
@@ -327,25 +327,33 @@
   "destinations.frequentlyUsed": "Most frequently used destinations",
   "destinations.dontHaveYourOwnDestination": "Don’t have your own destination yet?",
   "destinations.startWith": "Start with {name}",
-
+  
   "connection.stream.syncMode": "{syncMode} - {destinationSyncMode}",
 
   "destinationSyncMode.append": "Append",
   "destinationSyncMode.append_dedup": "Deduped + history",
   "destinationSyncMode.overwrite": "Overwrite",
-
+  
   "syncMode.full_refresh": "Full refresh",
   "syncMode.incremental": "Incremental",
-
-  "connection.emptyStateFooter": "or play around in our <demoLnk>demo instance</demoLnk>.",
+  
+  "connection.onboarding.title": "<b>Connections</b> link Sources to Destinations",
+  "connection.onboarding.createFirst": "Create your first connection",
+  "connection.onboarding.sources": "Sources",
+  "connection.onboarding.sourcesDescription": "A source is where Airbyte pulls data from. Airbyte supports hundreds of sources across APIs, databases, and files",
+  "connection.onboarding.moreSources": "Connect to one of {count}+ sources",
+  "connection.onboarding.addSource": "Connect to {source}",
+  "connection.onboarding.destinations": "Destinations",
+  "connection.onboarding.destinationsDescription": "A destination is where Airbyte sends data. These are typically data warehouses or databases.",
+  "connection.onboarding.addDestination": "Connect a source in order to send data to {destination}",
```

Comment:
Connect a source in order to send data to any of Airbyte's (count)+ destinations